components/CurrentPetStats.jsx

```diff
@@ -3,61 +3,70 @@ import StyledStatsContainer from "@/components/Styles/StyledStatsContainer";
 import StyledListItem from "@/components/Styles/StyledListItem";
 import StyledDiv from "@/components/Styles/StyledDiv";
 import PositionedButton from "@/components/Styles/StyledButton";
-
+import SVGIcon from "./SVGIcon";
 
 export default function CurrentPetStats({ selectedPet, onMode, isDead }) {
   return (
-      <StyledDiv >
-        <StyledStatsContainer>
-          <StyledListItem $variant="livingroom">
-            <StyledStatKey>Name</StyledStatKey>
-            <br />
-            {selectedPet.name}
-          </StyledListItem>
-          <StyledListItem $variant="livingroom">
-            <StyledStatKey>Type</StyledStatKey>
-            <br /> {selectedPet.type}
-          </StyledListItem>
-          <StyledListItem $variant="livingroom">
-            <StyledStatKey>Health</StyledStatKey>
-            <br /> {selectedPet.health}/100
-          </StyledListItem>
-          <StyledListItem $variant="livingroom">
-            <StyledStatKey>Happiness</StyledStatKey>
-            <br />
-            {selectedPet.happiness}/100
-          </StyledListItem>
-          <StyledListItem $variant="livingroom">
-            <StyledStatKey>Satiety</StyledStatKey>
-            <br />
+    <StyledDiv>
+      <StyledStatsContainer>
+        <StyledListItem $variant="livingroom">
+          <span>Name</span>
+          <br />
+          <StyledStatKey $variant="icon">{selectedPet.name}</StyledStatKey>
```

Comment:
Just out of curiosity, where do the <StyledStatsContainer> and <StyledStatKey> components come from? Because you haven't imported them like the other styled components.

Reply:
Maybe it was not shown on the changed files. But when you look at files changed, sometimes there is a "expand" option. which lets you look into lines of code which are not really touched the last time.
In this case the import of these two components was just not shown in the "files changed" on Github.